servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -740,10 +740,6 @@ void initObjectStore() {
             adminUsers.add(adminUser);
         }
         
-        if (!ZMSConsts.USER_DOMAIN.equals(userDomain)) {
```

Comment:
I think the bug here is not to remove this block. Instead we want to create both domains - userDomain and USER_DOMAIN.